CompuCell3D/core/CompuCell3D/steppables/PDESolvers/ReactionDiffusionSolverFVM.h

```diff
@@ -8,6 +8,7 @@
 
 #include <CompuCell3D/plugins/PixelTracker/PixelTracker.h>
 #include <CompuCell3D/plugins/PixelTracker/PixelTrackerPlugin.h>
+#include <CompuCell3D/plugins/NeighborTracker/NeighborTrackerPlugin.h>
```

Comment:
I think we may declare NeighborTrackingPlugin here:
namespace CompuCell3D{
   class NeighborTrackingPlugin
   ...

and we can move include to .cpp file  - minor issue though